src/grt/src/GlobalRouter.tcl

```diff
@@ -144,6 +144,18 @@ proc set_macro_extension { args } {
   }
 }
 
+sta::define_cmd_args "set_pin_offset" { offset }
```

Comment:
What is the purpose of this command?  Does it need to be documented or is just for developer use?

Reply:
It is just for developer use. It defines the offset (in gcells) that the fake pins will have from the original pin position.